### DISCUSSION TITLE
Feature field does not accepts the > (Greater than symbol) character

### DISCUSSION THREAD

**Discussion by musaffarpatel** [PrestaShop/PrestaShop#33843] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

When editing or adding a product and entering features under the details tab, it is not possible to enter text with characters such as > (greater than symbol).

### Steps to reproduce

Catalog > Products > Add Product
Under Details tab add a new feature
Enter text such as "Hello > World"

The following error is returned:

An unexpected error occurred. [PrestaShop\PrestaShop\Core\Domain\Feature\Exception\FeatureValueConstraintException code 20]

### PrestaShop version(s) where the bug happened

8.1

**Comment by kpodemski** [comment]
It seems like a correct behavior, not a bug. We've decided not to allow `>` in the feature name. 

This has to be consulted with @PrestaShop/product-council

**Comment by musaffarpatel** [comment]
@kpodemski Thanks, interesting decision as I would accept any description fields like this one to accept all alpha numeric characters.

**Comment by MatShir** [comment]
The Feature "entity" can't be saved with ">" character. The new page follows the rule of feature creation. I can't really say why it can't. I see as improvement since it has always "worked" that way

**Reply by MatShir** [reply]
We could do a bug report to better catch the error though
